routing/router_test.go

```diff
@@ -1025,6 +1025,77 @@ func TestIgnoreNodeAnnouncement(t *testing.T) {
 	}
 }
 
+// TestIgnoreChannelEdgePolicyForUnknownChannel checks that a router will
+// ignore a channel policy for a channel not in the graph.
+func TestIgnoreChannelEdgePolicyForUnknownChannel(t *testing.T) {
+	t.Parallel()
+
+	const startingBlockHeight = 101
+	ctx, cleanUp, err := createTestCtxFromFile(
```

Comment:
If you use `createTestGraphFromChannels`, you can create an empty graph and remove this (I think) unnecessary dependency on the test graph file.

Reply:
Fixed: https://github.com/lightningnetwork/lnd/pull/2549/commits/69ad94c68b696e79661fab322d6f0e90efdfb83d